src/utils.ts

```diff
@@ -8,7 +8,9 @@ export function groupDependenciesByManifest(
 ): Map<string, Changes> {
   const dependencies: Map<string, Changes> = new Map()
   for (const change of changes) {
-    const manifestName = change.manifest
+    // If the manifest is null or empty, use a space as the key to avoid
+    // breaking the HTML rendering later
+    const manifestName = change.manifest || ' '
```

Comment:
Rather than using invisible characters that could be confusing, maybe let's consider something like `'<no manifest specified>'` to make it clearer to the user?